manifests/JFLarvoire/Ag/2.2.3.yaml

```diff
@@ -0,0 +1,36 @@
+###############################################################################
+#                                                                             #
+#   Filename        ag.yaml                                                   #
+#                                                                             #
+#   Description     WinGet package manifest for The Silver Searcher           #
+#                                                                             #
+#   Notes                                                                     #
+#                                                                             #
+#   History                                                                   #
+#    2020-06-03 JFL Created this file.                                        #
+#                                                                             #
+###############################################################################
+
+Id: JFLarvoire.Ag
+Publisher: Jean-François Larvoire
+Name: The Silver Searcher
+Version: 2.2.3
+AppMoniker: Ag
+MinOSVersion: 5.1.0.0
+Description: A fast open source text search tool, optimized for software development
+Homepage: https://github.com/JFLarvoire/the_silver_searcher
+Tags: "find,grep"
+License: Apache Licence version 2.0
+LicenseUrl: https://github.com/JFLarvoire/the_silver_searcher/blob/master/LICENSE
+InstallerType: exe
+Switches:
+  Silent: -y
+  SilentWithProgress: -y
+Installers:
+  - Arch: x86
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
+  - Arch: x64
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
+ManifestVersion: 0.1.0
```

Comment:
I think the `ManifestVersion` field is not supported yet so you could remove this line in order to pass the manifest validation.

Reply:
OK, done, I've just pushed a change removing it.
Note that the command 'winget validate ...' did report "Manifest validation succeeded."
So there is an inconsistency between that and the bot validation code.